requirements.txt

```diff
@@ -1,4 +1,4 @@
-bokeh
+bokeh=1.4.0
```

Comment:
@twsearle this might need to be taken out for now until we officially move to the latest version of Bokeh

Reply:
done.